src/git/zcl_abapgit_git_pack.clas.abap

```diff
@@ -195,6 +195,8 @@ CLASS zcl_abapgit_git_pack IMPLEMENTATION.
 
     DO lv_objects TIMES.
 
+      DATA(uindex) = sy-index.
+
```

Comment:
Is this change in git decoding also related to optimizing of SAP packages?

Reply:
No. That is part of the improvement for object handling (#1606), that must have slipped through in my manual merge. Sorry. I'll fix it.